### ISSUE TITLE
Ant task broken

### ISSUE THREAD

**Issue by nic-gibson** [ndw/xmlcalabash1#157]
It looks like a recent version of calabash has misplaced the ant task. Looking inside the jar file shows the code still present but the XML file referenced from the docs (_com/xmlcalabash/antlib.xml_) isn't there any more. 

```
[nicg@newt-3 ~]$ ant
Buildfile: /Users/nicg/build.xml
    [typedef] Could not load definitions from resource com/xmlcalabash/antlib.xml. It could not be found.
```

**Comment by ndw** [comment]
Alex reports it still doesn't work. Back to the drawing board.

**Comment by ndw** [comment]
I believe this is fixed in XML Calabash 1.0.19, released today.

On further investigation, Alex's problem was something else, I think.
